tests/Doctrine/Tests/ORM/EntityNotFoundExceptionTest.php

```diff
@@ -30,4 +30,14 @@ public function testFromClassNameAndIdentifier()
         $this->assertInstanceOf(EntityNotFoundException::class, $exception);
         $this->assertSame('Entity of type \'foo\' was not found', $exception->getMessage());
     }
+
+    public function testNoIdentifierFound() : void
+    {
+        $exception = EntityNotFoundException::noIdentifierFound(
+            'foo',
+        );
```

Comment:
Trailing commas in method calls require php 7.3, and there is no need for that linebreak anyway IMO
```suggestion
        $exception = EntityNotFoundException::noIdentifierFound('foo');
```

Reply:
I'm fine with this.  However, this project needs to make up it's mind on what version is being targeted.  It's extremely confused.  `composer.json` says 7.3+.  Most of the code doesn't use typing either.

Reply:
The branch you're targeting to has `^7.2` as constraint, so 7.3 features are not usable.